### PR TITLE
Heatmap

### PR DESCRIPTION
**Pull request by dsctt**
Starting off PR for heatmap stuff. Depending on what's useful, further modifications can include different values instead of MSE like raw difference in V (or Q?) e.g. for certain obs.. as long as we can get scalars. (same goes for gradient stuff too).

### CODE REVIEW

File: grl/run.py [hunk 1]
```diff
@@ -70,32 +71,59 @@ def run_algos(spec, no_gamma, n_random_policies, use_grad, n_steps, max_rollout_
...
+            if (num_ticks * i + j + 1) % 10 == 0:
+                print(f'Calculating policy {num_ticks * i + j + 1}/{num_ticks * num_ticks}')
+
+    ax = sns.heatmap(np.array(discrepancies).reshape((num_ticks, num_ticks)),
+                     xticklabels=x.round(3),
+                     yticklabels=y.round(3))
+    ax.invert_yaxis()
+    ax.set(xlabel='2nd obs', ylabel='1st obs')
+    ax.set_title(args.spec)
+    plt.show()
```
Comment (camall3n) [inline grl/run.py:97]: Can you split this up to make things more abstract?

Like one part takes in a 2D array of heat values (and maybe some labels) and it just makes the plot for you. The other part computes that 2D array.

**Review by camall3n — CHANGES_REQUESTED** [review]

File: grl/run.py (hunk 1 above)
Comment (dsctt) [reply grl/run.py:97]: You mean into two functions?

Comment (camall3n) [reply grl/run.py:97]: Maybe that's overkill, now that I think about it more.

Comment (camall3n) [reply grl/run.py:97]: Never mind!

**Review by camall3n — APPROVED** [review]

File: grl/run.py (hunk 1 above)
Comment (dsctt) [reply grl/run.py:97]: Agree, it's good for now and we can abstract if we ever need some of the inner parts for something else.

**Pull request merged**